packages/data-point/lib/reducer-path/factory.test.js

```diff
@@ -35,3 +35,116 @@ describe('reducer/reducer-path#create', () => {
     expect(reducer.asCollection).toBe(true)
   })
 })
+
```

Comment:
These tests were moved over from lib/resolve/reducer-path.test.js

Reply:
Very nice! These tests illustrate the benefit of moving this to the new `reducer.body` method well.